.github/workflows/build.yaml

```
@@ -31,12 +28,22 @@ jobs:
         run: |
           which python
           python --version
+      - name: Install Fortran compiler on Ubuntu
+        if: matrix.os == 'ubuntu-latest'
+        run: sudo apt-get update && sudo apt-get install -y gfortran
+      - name: Install Fortran compiler on macOS
+        if: matrix.os == 'macos-latest'
+        run: brew install gcc
```

```suggestion
        run: |
          brew install gcc@14
          echo "FC=gfortran-14" >> "$GITHUB_ENV
```

This makes the fortran compiler available through the FC variable, if this works, I think it is the cleanest option